com.espindola.lobwebapp.exception.lobwebappexception=Houve um erro interno no sistema
com.espindola.lobwebapp.exception.alreadyexistsexception=Erro item já existente
com.espindola.lobwebapp.exception.notfoundexception=Erro item não encontrado
com.espindola.lobwebapp.exception.invalidargumentexception=Erro argumento inválido

com.espindola.lobwebapp.exception.productalreadyexistsexception=Produto já existente: {0}
com.espindola.lobwebapp.exception.productinvalidexception=Produto inválido: {0}
com.espindola.lobwebapp.exception.productnotfoundexception=Produto não encontrado com o valor: {0}

com.espindola.lobwebapp.exception.customeralreadyexistsexception=Cliente já existente: {0}
com.espindola.lobwebapp.exception.customerinvalidexception=Cliente inválido: {0}
com.espindola.lobwebapp.exception.customernotfoundexception=Cliente não encontrado com o valor: {0}

com.espindola.lobwebapp.exception.orderalreadyexistsexception=Pedido já existente: {0}
com.espindola.lobwebapp.exception.orderinvalidexception=Pedido inválido: {0}
com.espindola.lobwebapp.exception.ordernotfoundexception=Pedido não encontrado com o valor: {0}

com.espindola.lobwebapp.exception.stockalreadyexistsexception=Estoque já existente: {0}
com.espindola.lobwebapp.exception.stockinvalidexception=Estoque inválido: {0}
com.espindola.lobwebapp.exception.stocknotfoundexception=Estoque não encontrado com o valor: {0}

com.espindola.lobwebapp.exception.useralreadyexistsexception=Usuário já existente: {0}
com.espindola.lobwebapp.exception.userinvalidexception=Usuário inválido: {0}
com.espindola.lobwebapp.exception.usernotfoundexception=Usuário não encontrado com o valor: {0}